src/main/webapp/i18n/en/instructorDashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+    "artemisApp": {
+        "instructorDashboard": {
+            "buildPlans": {
+                "delete": {
+                    "question": "Are you sure you want to delete build plans for Exercise {{ id }}?"
```

Comment:
Please change to "Are you sure you want to delete all student build plans for the Programming Exercise"
Instead of id, can you show the name of the exercise and the course it belongs to?

Reply:
Sure, done.